src/CheckoutSdk/Sources/SourceResponse.cs

```diff
@@ -0,0 +1,47 @@
+using Checkout.Common;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Checkout.Sources
+{
+    /// <summary>
+    /// Defines a source response.
+    /// </summary>
+    public class SourceResponse : Resource
+    {
+        /// <summary>
+        /// Gets or sets the processed response returned following a successfully processed source (HTTP Status Code 201).
+        /// </summary>
+        public SourceProcessed Source { get; set; }
+
+        /// <summary>
+        /// Gets or sets the pending response returned for asynchronous source or when further action such as a redirect is required (HTTP Status Code 202).
+        /// </summary>
+        public SourcePending Pending { get; set; }
```

Comment:
Not supported though I think it's a good idea to leave this wrapper type in as it's possible we will have redirect sources in the future